src/bot.py

```diff
@@ -173,6 +144,7 @@ async def load_passwords(self, guild: discord.Guild) -> AsyncGenerator[Tuple[str
         """
         message: discord.Message
         async for message in self.passwords_channel.history(limit=100, oldest_first=True):
-            content: str = message.content.replace('`', '').strip()
-            team, password = content.split(':')
-            yield team.strip(), password.strip()
+            if message.content.startswith('```'):
+                content: str = message.content.replace('`', '').strip()
+                team, password = content.split(':')
+                yield team.strip(), password.strip()
```

Comment:
passwords and TLAs are assumed lower case, so for safety you should call `.lower()` here to avoid bugs from bad input
```suggestion
                yield team.lower().strip(), password.lower().strip()
```

Reply:
TLAs are now always upper case
https://github.com/srobo/discord-bot/pull/31/files#diff-68a0ee316db60f5fe20d546564b5d5cf65a066e4cf6a1ad0c97eec472d0e72f0R91